app/models/game.rb

```diff
@@ -87,7 +88,8 @@ def self.search(search)
     if Utilities.BARCODE_FORMAT.match(search) && !/[a-z]+/.match(search) && /\d+/.match(search)
       result = where(barcode: search.upcase)
     elsif search
-      result = where(title: Title.search(search))
+      search_query = "%#{search}%"
+      result = joins(:title).joins(:publisher).where('titles.title ILIKE ? OR publishers.name ILIKE ?', search_query, search_query)
```

Comment:
This is an ok option, but let's not add in publisher searching like this. Adding a fixed OR statement is likely to decrease performance (we want to keep searches as fast as possible), and it makes it unclear what part of the game is being searched. I've added story #53 to cover what we should update on the search page.